test.js

```diff
@@ -1,5 +1,12 @@
-var t = "hello";
+  var t = "hello";
+var b = null
 
-if (t === "hello') {
-    console.log('ok');
+if (t == "hello") {
+ console.log('ok');
 }
+
+if (true) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2496460053/issues/source?bid=3260975&fileBranchId=3269370#l8)